index.html

```diff
@@ -861,8 +857,8 @@ <h2>
       <p>
         We would like to offer our sincere thanks to Mounir Lamouri, Sergey
         Konstantinov, Matvey Larionov, Dominique Hazael-Massieux, Domenic
-        Denicola, Thomas Steiner, Raphael Kubo da Costa for their contributions
-        to this work.
+        Denicola, Thomas Steiner, Raphael Kubo da Costa, Anne van Kesteren for
```

Comment:
No need to thank an editor ;) 

```suggestion
        Denicola, Thomas Steiner, Anne van Kesteren for
```

Reply:
Done :-)